PhoneNumberKit/UI/TextField.swift

```diff
@@ -34,10 +34,12 @@ open class PhoneNumberTextField: UITextField, UITextFieldDelegate {
         super.text = newValue
     }
 
+    private lazy var _defaultRegion: String = PhoneNumberKit.defaultRegionCode()
+
     /// Override region to set a custom region. Automatically uses the default region code.
-    open var defaultRegion = PhoneNumberKit.defaultRegionCode() {
-        didSet {
-            partialFormatter.defaultRegion = defaultRegion
+    open var defaultRegion: String {
+        get {
```

Comment:
so this new code:
(a) makes it so that any implementor of PhoneNumberKit who had a `textField.defaultRegion = GB` means that now the partialFormatter associated with the TextField will no longer have `partialFormatter.defaultRegion` set to the new value, and so the partialFormatter will continue working with the old defaultRegion.  that seems like a breaking change to me.
(b) the new getter doesn't really appear to do anything that a get from a call to `textField.defaultRegion` would have used to do.  after all, the new `_defaultRegion` is private, and is set to what the old var would have used to have been set to.  i'm not seeing the reason for self._defaultRegion here.

Reply:
(a) I find usage of `defaultRegion` only in `init`, what is the setter for? or didn’t I see?

(b) `defaultRegion` must be computed, so that it can be overridden, `_defaultRegion` is used for lazy optimization if someone wants to access `defaultRegion` many times and if `defaultRegion` was not overridden.

Also it can be like:
```
open var defaultRegion: String {
            return PhoneNumberKit.defaultRegionCode()
    }
```
but it will be calculated every time when accessed by a developer.